playbooks/nfd_operator_status_check.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Check the status of various NFD resources
```

Comment:
"Check the status" could be a bit ambiguous. Is the purpose to make sure the status is ok and explicitly fail if it's not ok, or is the purpose to dump and save the status of all resources for diagnostic purposes?